tests/.infra/crw-ci/nightly/k8s/single-host/Jenkinsfile

```diff
@@ -164,4 +173,4 @@ pipeline {
             }
```

Comment:
Could you, please, add next code into `post {` section to send the mail in case of successful execution as well:
```
        success {
            script {
                if (emails != "") {
                    mail to: emails,
                            subject: "Jenkins Job $currentBuild.fullDisplayName has passed successfully.",
                            body: "Build $currentBuild.absoluteUrl has passed successfully."
                }
            }
        }
```

Reply:
Done.

Reply:
Thank you!